test.py

```diff
@@ -0,0 +1,10 @@
+x = "\x1f"
```

Comment:
stray file?

Reply:
Yep, forgot to remove it before commiting.